### PR TITLE
Add some shortcode infos to contribute.md

### PR DESCRIPTION
**Pull request by fkaminski**
For reminder ...

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,18 +27,82 @@ Cum sociis [natoque](https://www.google.com/) penatibus et magnis dis.
 ```
 
 
+### Short code notice
+
+The notice shortcode shows 4 types of disclaimers.
```
Comment (fritzmg) [inline CONTRIBUTING.md]:
```suggestion
The `notice` shortcode allows you to display 4 different types of disclaimers.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -27,18 +27,82 @@ Cum sociis [natoque](https://www.google.com/) penatibus et magnis dis.
...
+```
+{{% notice warning %}}
+A warning disclaimer
+{{% /notice %}}
+```
+
+
+### Short code tabs
+
+Choose which content to see across the page. Very handy for providing code snippets for multiple languages or
```
Comment (fritzmg) [inline CONTRIBUTING.md]: > Choose which content to see across the page.

Not quite sure what you mean by that? The short code tab allows you to group content into tabs.

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (fkaminski) [reply CONTRIBUTING.md]: Just using the original description from https://learn.netlify.app/en/shortcodes/tabs/
Will fix that ...

Comment (fkaminski) [reply CONTRIBUTING.md]: Done

**Review by fritzmg — APPROVED** [review]

**Comment by fritzmg** [comment]
thank you @fkaminski :)

**Pull request merged**